server/src/test/java/org/spine3/server/projection/ProjectionShould.java

```diff
@@ -73,13 +73,19 @@ public void throw_exception_if_no_handler_for_event() {
 
     @Test
     public void return_event_classes_which_it_handles() {
-        final ImmutableSet<EventClass> classes = Projection.getEventClasses(TestProjection.class);
+        final ImmutableSet<EventClass> classes =
+                Projection.TypeInfo.getEventClasses(TestProjection.class);
 
         assertEquals(TestProjection.HANDLING_EVENT_COUNT, classes.size());
         assertTrue(classes.contains(EventClass.of(StringValue.class)));
         assertTrue(classes.contains(EventClass.of(Int32Value.class)));
     }
 
+    @Test
+    public void have_type_info_utility_class() {
+        assertHasPrivateParameterlessCtor(Projection.TypeInfo.class);
```

Comment:
The same case again. BTW, this test case has another name which may be more appropriate for all the test cases like this.